### ISSUE TITLE
Autofill not working 

### ISSUE THREAD

**Issue by Xisrr1** [AChep/keyguard-app#307]
I've tried on different apps and websites and Keyguard will not prompt to save a password while google password manager will. Even when adding the password manually it won't autofill it and I will need to open the vault and select manually the password.

**Comment by AChep** [comment]
Did you enable Keyguard as an autofill provider in the Settings / Autofill / Autofill service? If yes, what browser / apps did you test it later on?

**Comment by Xisrr1** [comment]
It doesn't even ask to save the password for Discord.

**Comment by Gigno91** [comment]
Same on my Samsung S23 Plus.
Keyguard as default autofill provider, with all permission granted, but nor the prompt (sometimes only) or the inline keyboard suggestion appears.

**Comment by AChep** [comment]
> Same on my Samsung S23 Plus. Keyguard as default autofill provider, with all permission granted, but nor the prompt (sometimes only) or the inline keyboard suggestion appears.

What browser are you testing it with?

**Comment by Gigno91** [comment]
> > Same on my Samsung S23 Plus. Keyguard as default autofill provider, with all permission granted, but nor the prompt (sometimes only) or the inline keyboard suggestion appears.
> 
> What browser are you testing it with?

Latest Kiwi Browser.
Sometimes the prompt suggestion appear, but with no logic behind that i can understand, even for inline keyboard suggestion with both Gboard or Samsung Keyboard.

**Comment by Xisrr1** [comment]
Yep

**Comment by Xisrr1** [comment]
Is it fixed with the latest update?

**Comment by Gigno91** [comment]
Yes, the issue seems to be fixed with Kiwi and Chrome, but when you try to save a passkey only the Google Password Manager will prompt you to save the generated passkey, even if Keyguard is the only password manager setted up.
Original Bitwarden seems to work saving passkeys instead.

**Comment by Gigno91** [comment]
After using a bit this app, maybe i'm wrong, but i figured out that it works better with Cromite and every other browser that supports the native Android autofill implementation.
Even Chrome with in-app autofill disabled shows the Google password prompt (wtf?)...
In theory, the popup prompt should work with apps that doesn't support the native autofill service, but sometimes appears and sometimes it doesn't.
Internet browsers like Samsung Internet, Chrome itself or any other browser doesn't like Keyguard at all, but i suggest to use the android accessibility service to make stronger the Keyguard autofill in those "problematic" apps, like original (xamarin) Bitwarden does.

Hope my comments help...
